src/emit/mod.rs

```diff
@@ -5547,32 +5855,48 @@ pub trait TargetRuntime<'a> {
         let bits = left.get_type().get_bit_width();
 
         if bits > 64 {
-            // round up the number of bits to the next 32
+            // Round up the number of bits to the next 32
             let mul_bits = (bits + 31) & !31;
             let mul_ty = bin.context.custom_width_int_type(mul_bits);
 
-            // round up bits
+            // Round up bits
             let l = bin.build_alloca(function, mul_ty, "");
             let r = bin.build_alloca(function, mul_ty, "");
             let o = bin.build_alloca(function, mul_ty, "");
 
             if mul_bits == bits {
                 bin.builder.build_store(l, left);
                 bin.builder.build_store(r, right);
-            } else if signed {
-                bin.builder
-                    .build_store(l, bin.builder.build_int_s_extend(left, mul_ty, ""));
-                bin.builder
-                    .build_store(r, bin.builder.build_int_s_extend(right, mul_ty, ""));
-            } else {
+            }
+            // LLVM-IR can handle multiplication of sizes up to 64 bits. If the size is larger, we need to implement our own mutliplication function.
+            // We divide the operands into sizes of 32 bits (check __mul32 in stdlib/bigint.c documentation).
+            // If the size is not divisble by 32, we extend it to the next 32 bits. For example, int72 will be extended to int96.
+            // Here, We zext the operands to the nearest 32 bits. zext is called instead of sext because we need to do unsigned multiplication by default.
```

Comment:
```suggestion
            // Here, we zext the operands to the nearest 32 bits. Zext is called instead of sext because we need to do unsigned multiplication by default.
```
Incorrect usage of capital letters.